server/src/com/thoughtworks/go/server/service/ArtifactsService.java

```diff
@@ -390,7 +401,7 @@ private boolean deleteFile(File file) {
     }
 
     public void appendToConsoleLog(JobIdentifier jobIdentifier, String text) throws IllegalArtifactLocationException, IOException {
-        File file = findArtifact(jobIdentifier, ArtifactLogUtil.getConsoleLogOutputFolderAndFileName());
```

Comment:
I think it's time to start thinking about a ```ConsoleArtifactService```. This ```ArtifactService``` is too big, and some methods like ```findConsoleArtifact``` and ```temporaryConsoleFile``` are too confusing to tell apart. However, I won't hold this PR back because of this.